app/src/main/res/values-de/strings.xml

```diff
@@ -84,11 +84,11 @@
     <string name="decline_button_cnt_des">Transfer ablehnen</string>
     <string name="accept_button_cnt_des">Transfer annehmen</string>
     <string name="stop_button_cnt_des">Transfer anhalten</string>
-    <string name="retry_button_cnt_des">Transfer erneut versuchen</string>
+    <string name="retry_button_cnt_des">Transfer erneut starten</string>
     <!--  Other  -->
     <string name="please_select_download_dir">Dies ist das erste Mal, dass Sie die Anwendung starten.
         Bitten wählen Sie ein Verzeichnis, in dem Warpinator Dateien speichern soll.</string>
-    <string name="error_download_dir">Sie müssen in den Einstellungen ein Downloadverzeichnis auswählen</string>
+    <string name="error_download_dir">Sie müssen in den Einstellungen ein Downloadverzeichnis auswählen.</string>
```

Comment:
uDEV2019 wrote, --> sphenotux comments
Line 91 has in the source no punctuation mark. --> Maybe, OK - in German, this is a whole sentence, and a sentence ends with a point. I guess the point is missing in the English source by error?